modules/nats/options.go

```diff
@@ -49,3 +50,22 @@ func WithArgument(flag string, value string) CmdOption {
 		o.CmdArgs[flag] = value
 	}
 }
+
+// WithConfigFile pass io.Reader to the NATS container as /etc/nats.conf
```

Comment:
suggestion: as above clarify that reader is used to create a file not passed as the arg directly as that's confusing to the reader

Reply:
fixed in ff17ba124cb6cf9c72bfd04206630fd4afb6fb8a